### DISCUSSION TITLE
Export config file

### DISCUSSION THREAD

**Discussion by Spo0on** [noah-nuebling/mac-mouse-fix#236] [Questions] [answered]
Is there a way to export configurations in a config file?

**Comment by noah-nuebling** [comment, marked as answer]
Hi @Clejd!

There's no 'official' way to do this, but I've secretly built some rudimentary support for exporting / importing configurations for development purposes.

Here's how to do it:
Go to the 'About' tab (or the 'More...' screen, if you’re on MMF 2 or below), then Click the MMF Logo while holding Command (⌘) and Shift (⇧). This will reveal the 'config.plist' file in Finder.

The 'config.plist' file holds all your settings, including button remappings, scroll settings, etc.

You can save this 'config.plist' file somewhere else and you can also replace it with another config.plist file you've saved before. This will allow you to import/export configurations.

Hope that helps! Let me know if you have any more questions.

**Reply by vii33** [reply]
@noah-nuebling  Thank you for the reply. I would be great if the file location could be mentioned in the github readme so people can find it easier. Also a "standard link" in the "About" Section (instead of a hidden keybinding) would be great. I would guess this is helpful for many people, no need to hide it.
